src/components/calendar.vue

```diff
@@ -118,13 +119,13 @@
           changeMonth: function (monthStart) {
             this.currentMonthStart = monthStart;
             let monthEnd = dateHelper.lastDateOfMonth(new Date(monthStart));
-	          this.$emit('monthChanged', monthStart, monthEnd);
+            this.$emit('month-changed', monthStart, monthEnd);
           },
           dayClick: function (day) {
-            this.$emit('dayClicked', day);
+            this.$emit('day-clicked', day);
           },
-          eventClicked: function(event) {
-          	this.$emit('eventClicked', event);
+          eventClicked: function(event, day) {
```

Comment:
@Trekels  plz, pay attention to this change

Reply:
@fobiasmog I'll keep in mind, but these will be temporary fixes since im working on a cleaner V1 that is more flexible and some of these events wil be very different